app/controllers/questionable/answers_controller.rb

```diff
@@ -2,6 +2,8 @@
 
 module Questionable
   class AnswersController < ApplicationController
+    skip_before_action :require_full_user!
```

Comment:
Will this be problematic for others using this gem since the method will not exist?